lib/tasks/publishing_api.rake

```diff
@@ -128,6 +128,29 @@ namespace :publishing_api do
     PublishingApiDocumentRepublishingWorker.new.perform(document.id)
   end
 
+  desc "Manually unpublish content with a redirect"
+  # This task is for unpublishing Whitehall managed content, where
+  # Whitehall has forgotten it is managing the content (as it often
+  # does).
```

Comment:
Could you rewrite the comment to say when you expect to use this? eg

```ruby
# This task is for unpublishing and redirecting Whitehall content. Normally,
# unpublishing deletes the content, making it difficult to set up a redirect.
# Do not use this task if the content has already been deleted - you'll need to
# do something else for that (TBC).
```

Reply:
To me, that is when Whitehall has forgotten it's managing the content in question.

Luckily, this isn't a normal situation, for most content managed by Whitehall, it isn't deleted from the database when you unpublish it.

I've changed this now though, is this any better?

```
  # This task is for unpublishing Whitehall managed content where
  # Whitehall has forgotten it is managing the content (as it often
  # does). Do not use this Rake task for content which Whitehall still
  # has a record of.
```